lib/identity/auth.rb

```diff
@@ -245,6 +244,20 @@ def redirect_or_authorize
       end
     end
 
+    def redirect_to_sso?
+      # If the user uses SSO and SSO is configured in Identity, continue on to
+      # check if their access token is valid.
+      return false unless @cookie.sso_entity && Config.sso_base_url
+      return true if !@cookie.access_token
```

Comment:
Better to use "unless" instead of "if !" here? Like:

`return true unless @cookie.access_token.present?`